Source/astcenc_vecmathlib.h

```diff
@@ -0,0 +1,309 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
General comment - can you rebase. I fixed an unrelated bug which was tripping our HDR tests, and I'd like to include that fix in the tests for this one before I merge it.